modules/webgl/src/adapter/converters/device-parameters.ts

```diff
@@ -278,9 +278,17 @@ export function setDeviceParameters(device: Device, parameters: Parameters) {
   // },
 
   // COLOR STATE
+  if (parameters.blend) {
+    gl.enable(GL.BLEND);
+  } else if (parameters.blend === false) {
+    gl.disable(GL.BLEND);
+  }
```

Comment:
I would type out the 3 cased more clearly
```suggestion
  if (parameters.blend === true) {
    gl.enable(GL.BLEND);
  } else if (parameters.blend === false) {
    gl.disable(GL.BLEND);
  }
  // leave unchanged if `parameters.blend` is not set
```